content/about.md

```diff
@@ -13,10 +13,17 @@ weight: 1
 
 ---
 
-**Registration is closed. Thank you for shaping an exciting event with your contribution!**
-
-We are currently creating a schedule, and will send out acceptance emails by November 1st.
-If you’ve missed the deadline but wanted to participate, please stay tuned - we will be able to accommodate late registrations in a short while.
+**Registration Notifications and Late Submissions**
```

Comment:
```suggestion
**Registration Notifications and Late Registrations**
```

I get that the duplication of "registration" is annoying, but "late submission" sounds like we would still accept talks

Reply:
I agree with @adswa. Alternatively, these can be called "Passive registration".